opentreemap/treemap/tests/test_udfs.py

```diff
@@ -1061,10 +1061,8 @@ def test_string_datatype(self):
         self._test_datatype('Test string', 'Sweet Plot')
 
     def test_in_operator(self):
-        self.assertEqual('Test string' in self.plot.udfs,
-                         True)
-        self.assertEqual('RanDoM NAme' in self.plot.udfs,
-                         False)
```

Comment:
It turns out that I had misunderstood the semantics of the old `DataDictionary` based `UDFDictionary` for `in` and `iteritems`. I thought it was `True` for all udf definition names, but it was only `True` for all collection names. (Also see changes in `udf.py` for `__contains__` and `iteritems`.)